server/service/answer.rs

```diff
@@ -242,6 +241,9 @@ fn encode_date_time(date_time: NaiveDateTime) -> typedb_protocol::value::Datetim
     typedb_protocol::value::Datetime { seconds: date_time.and_utc().timestamp_millis(), nanos: date_time.nanosecond() }
 }
 
-fn encode_time_zone(timezone: Tz) -> typedb_protocol::value::datetime_tz::Timezone {
-    typedb_protocol::value::datetime_tz::Timezone::Named(timezone.name().to_string())
+fn encode_time_zone(timezone: TimeZone) -> typedb_protocol::value::datetime_tz::Timezone {
+    match timezone {
+        TimeZone::IANA(tz) => typedb_protocol::value::datetime_tz::Timezone::Named(tz.name().to_string()),
+        TimeZone::Fixed(fixed) => typedb_protocol::value::datetime_tz::Timezone::Offset(fixed.local_minus_utc()),
```

Comment:
Note: protocol should probably specify that the offset is in seconds. @flyingsilverfin